upb/arena.c

```diff
@@ -69,6 +69,24 @@ static upb_Arena* arena_findroot(upb_Arena* a) {
   return a;
 }
 
+size_t upb_Arena_SpaceAllocated(upb_Arena *arena) {
+  arena = arena_findroot(arena);
+  size_t memsize = 0;
+
+  mem_block* block = arena->freelist;
+
+  while (block) {
+    memsize += sizeof(mem_block) + block->size;
+    block = block->next;
+  }
+
+  return memsize;
+}
+
+uint_32_t upb_Arena_DebugRefCount(upb_Arena *arena) {
```

Comment:
Sorry, I mistyped before, this should be just `uint32_t` (not `uint_32_t`).

Reply:
No worries, I should have caught it.